firmware/ja-rule.X/nbproject/configurations.xml

```diff
@@ -31,38 +31,42 @@
         <itemPath>../src/logger_private.h</itemPath>
         <itemPath>../src/flags.h</itemPath>
         <itemPath>../src/usb_transport.h</itemPath>
+        <itemPath>../src/syslog.h</itemPath>
+        <itemPath>../src/usb_console.h</itemPath>
       </logicalFolder>
       <logicalFolder name="bsp" displayName="bsp" projectFiles="true">
         <logicalFolder name="pic32mx_eth_sk2"
                        displayName="pic32mx_eth_sk2"
                        projectFiles="true">
-          <itemPath>/opt/microchip/harmony/v1_02/bsp/pic32mx_eth_sk2/bsp_config.h</itemPath>
+          <itemPath>../../../../../../opt/microchip/harmony/v1_02/bsp/pic32mx_eth_sk2/bsp_config.h</itemPath>
         </logicalFolder>
       </logicalFolder>
       <logicalFolder name="framework" displayName="framework" projectFiles="true">
         <logicalFolder name="driver" displayName="driver" projectFiles="true">
           <logicalFolder name="usb" displayName="usb" projectFiles="true">
-            <itemPath>/opt/microchip/harmony/v1_02/framework/driver/usb/drv_usb.h</itemPath>
+            <itemPath>../../../../../../opt/microchip/harmony/v1_02/framework/driver/usb/drv_usb.h</itemPath>
           </logicalFolder>
         </logicalFolder>
         <logicalFolder name="system" displayName="system" projectFiles="true">
           <logicalFolder name="common" displayName="common" projectFiles="true">
-            <itemPath>/opt/microchip/harmony/v1_02/framework/system/common/sys_common.h</itemPath>
-            <itemPath>/opt/microchip/harmony/v1_02/framework/system/common/sys_module.h</itemPath>
+            <itemPath>../../../../../../opt/microchip/harmony/v1_02/framework/system/common/sys_common.h</itemPath>
```

Comment:
This, and all other changes in the file need to be moved back to absolute paths.